torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp

```diff
@@ -94,6 +97,12 @@ enum ErrorHandlingMode {
 
 #define SHOULD_TEAR_DOWN(a) (a != NoHandling && a != CleanUpOnly)
 
+#define PRINT_COLLECTIVE_HASH_SIGNATURE(                                    \
+    prefix, phase, opType, numel, hashValue)                                \
+  LOG(WARNING) << prefix << "at phase " << phase << ": Collective hash of " \
```

Comment:
could put the `logPrefix()` call in here and not require prefix arg

and simplify message to someting like 

"Hash of " << phase << " to NCCL" << opType << " with size " << numel << " was " << hashValue;
e.g.
"Hash of input to NCCL allreduce with size 256 was 0x123123123"

Reply:
> could put the `logPrefix()` call in here and not require prefix arg

Let me try.